bin/build_dependencies.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+#
+# Build local installs of python-flint's dependencies.
+# This should be run after bin/download_dependencies.sh
+#
+
+set -o errexit
+set -o xtrace
+
+# Sets variables like PREFIX and dependency versions
+source bin/build_variables.sh
+
+cd $PREFIX
+mkdir -p src
+cd src
+
+curl -O https://gmplib.org/download/gmp/gmp-$GMPVER.tar.xz
+curl -O https://www.mpfr.org/mpfr-current/mpfr-$MPFRVER.tar.gz
+curl -O https://www.flintlib.org/flint-$FLINTVER.tar.gz
+curl -O -L https://github.com/fredrik-johansson/arb/archive/refs/tags/$ARBVER.tar.gz
+mv $ARBVER.tar.gz arb-$ARBVER.tar.gz
+
+tar xf mpfr-$MPFRVER.tar.gz
+tar xf flint-$FLINTVER.tar.gz
+tar xf arb-$ARBVER.tar.gz
+tar xf gmp-$GMPVER.tar.xz
+
+cd gmp-$GMPVER
+  ./configure --prefix=$PREFIX --enable-fat --enable-shared=yes --enable-static=no
```

Comment:
This needs to add `--host=x86_64-unknown-linux-gnu` or `--host=x86_64-apple-darwin` to not use newer instructions. (Currently I see `broadwell-pc-linux-gnu` in the github actions logs which is not what we want)

Reply:
Thanks for pointing this out!

I think it's fixed in #17. Does that look okay to you?